### PR TITLE
fix: Avatar group size

### LINKED ISSUE #27530: Avatar group size
### Reproduction link
[https://ant.design/components/avatar-cn/#components-avatar-demo-group](https://ant.design/components/avatar-cn/#components-avatar-demo-group)

| Environment | Info |
|---|---|
| antd | 4.8.0 |

- https://github.com/ant-design/ant-design/pull/27348

嵌套 Tooltip 的 Avatar size 没有生效，观察第三个。

### PR DESCRIPTION
**Pull request by hengkx**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
close #27530

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   fix Avatar.Group size invalid.        |
| 🇨🇳 Chinese |   修复 Avatar.Group 的 size 不生效问题。        |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by xrkffgg** [comment]
我看还是小的？我拉到本地看看

**Comment by xrkffgg** [comment]
用法变了？

**Comment by hengkx** [comment]
没变吧

**Comment by xrkffgg** [comment]
你那看着 size 生效了吗？

**Comment by hengkx** [comment]
我脑残了

File: components/avatar/SizeContext.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { ScreenSizeMap } from '../_util/responsiveObserve';
+
+export type AvatarSize = 'large' | 'small' | 'default' | number | ScreenSizeMap;
+
+const SizeContext = React.createContext<AvatarSize>('default');
+
+export interface SizeContextProps {
+  size?: AvatarSize;
+}
```
Comment (xrkffgg) [inline components/avatar/SizeContext.tsx:10]:
```suggestion
};
```

**Review by xrkffgg — APPROVED** [review]

File: components/avatar/SizeContext.tsx (hunk 1 above)
Comment (hengkx) [reply components/avatar/SizeContext.tsx:10]: 自动格式化的 不用吧

**Pull request merged**